js/private/js_binary.bzl

```diff
@@ -267,8 +267,7 @@ _ATTRS = {
         attribute based on a `config_setting` rule.
         See /js/private/BUILD.bazel in rules_js for an example.
         """,
-        # TODO(2.0): make this mandatory so that downstream binary rules that inherit these attributes are required to set it
```

Comment:
There are some `hasattr(ctx.attr, "unresolved_symlinks_enabled")` calls that we can maybe delete at the same time?

Reply:
They all get deleted further up the stack so it will be easier to unwind the stack by just continuing down it